### ISSUE TITLE
Visible focus for Mobile

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4364]
Convert all visible focus desktop styles to their best equivalent for mobile.

**Considerations for Mobile Accessibility:**
https://cxwiki.sap.com/display/spar/Mobile+Accessibility

**Comment by developpeurweb** [comment]
After a first testing round in a real iPhone 7 device the following bugs were detected:

- Some elements are missing the focus indicator on iPhone (on menu links, forgot password link, footer links, checkboxes, radio buttons, buttons) 
- Modal's close icon has a partially hidden focus indicator on iPhone 
- Coupons button hides focus indicator on iPhone
- Skip link doesn’t show up on iPhone

I'm closing this ticket and creating specific ones for each issue.
